src/templates/home-page.js

```diff
@@ -25,24 +26,41 @@ export const HomePageTemplate = ({
   advisors_section: advisorsSection = {},
   supporters_section: supportersSection = {},
 }) => {
-  const handleSignupDiscord = ({ email }) => {
-    signupMailingList({ email, discordSignup: true })
-  }
-
   return (
     <div className="main-content">
-      <PageSection id={sections.HOME}>
+      <PageSection
+        id={sections.HOME}
+        style={{
+          backgroundImage: `url(${hero.bg_image.childImageSharp.fluid.src})`,
```

Comment:
I'll admit that I would prefer to replace the image file with a set filename rather than set the style property directly here, fwiw.

Reply:
Addressed in c0f84bb!